diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt

```diff
@@ -77,7 +77,7 @@ enum class Warnings(
     HEADER_NOT_BEFORE_PACKAGE(true, "2.2.1", "header KDoc should be placed before package and imports"),
     COMMENTED_OUT_CODE(false, "2.4.2", "you should not comment out code, use VCS to save it in history and delete this block"),
     WRONG_NEWLINES_AROUND_KDOC(true, "2.4.1", "there should be a blank line above the kDoc and there should not be no blank lines after kDoc"),
-    FIRST_COMMENT_NO_SPACES(true, "2.4.1", "there should not be any spaces before first comment"),
+    FIRST_COMMENT_NO_BLANK_LINE(true, "2.4.1", "there should not be any blank lines before first comment"),
```

Comment:
What does it check once again? Is this a first comment in the file? How is this different from blank lines rule?

Reply:
It checks that first comment starts with new line. BlankLinesRule handles blank lines in blocks before/after braces. Seems like blank lines before first comment have more complicated logic.

Reply:
ok then